### PR TITLE
bump to version 3.2.0 for PR #21

### PR DESCRIPTION
**Pull request by colinsurprenant**
Bump to version 3.2.0.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
+## 3.2.0
+  - Encode tags as fluent forward protocol tags
```
Comment (jakelandis) [inline CHANGELOG.md]: minor: If there is a corresponding issue, it is nice to like to it with #num convention

**Comment by jakelandis** [comment]
LGTM (minor optional suggestion)

File: CHANGELOG.md (hunk 1 above)
Comment (colinsurprenant) [reply CHANGELOG.md]: Aye - yes, good catch.

**Comment by colinsurprenant** [comment]
@jakelandis updated, thanks! will go ahead and merge + publish.

**Pull request merged**